index.js

```diff
@@ -52,12 +52,54 @@ function ownEnumerableKeys(obj) {
 	return res;
 }
 
+/**
+ * Determine the iterator to use, based on the given options.
+ * Switches between `ownEnumerableKeys` and `objectKeys`.
+ * Given includeSymbols, `ownEnumerableKeys` is used and iterates over symbols.
+ * Otherwise, `objectKeys` is used and iterates over strings only.
+ * @param {Object} options
+ * @returns {typeof ownEnumerableKeys | typeof objectKeys}
+ */
+function determineIterator(options) {
+	var opts = options || {};
+	return opts.includeSymbols === true ? ownEnumerableKeys : objectKeys;
+}
+
+/**
+ * Strip symbols from an array based on the given options.
+ * @param {Array} arr
+ * @param {Object} options
+ * @returns {typeof arr}
+ */
+function stripSymbols(arr, options) {
```

Comment:
I’m confused, why would we need to strip them when we could avoid adding them in the first place?